<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout">
        <item>
          <layout class="QHBoxLayout">
            <item>
              <widget class="QLineEdit" name="nameEdit">
                <property name="placeholderText">
                  <string>Название задачи</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="descEdit">
                <property name="placeholderText">
                  <string>Описание</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QDateTimeEdit" name="dateTimeEdit"/>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout">
            <item>
              <widget class="QPushButton" name="addButton">
                <property name="text">
                  <string>Добавить</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="deleteButton">
                <property name="text">
                  <string>Удалить</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QTableWidget" name="taskTable">
            <column>
              <property name="text">
                <string>Название</string>
              </property>
            </column>
            <column>
              <property name="text">
                <string>Описание</string>
              </property>
            </column>
            <column>
              <property name="text">
                <string>Дата и время</string>
              </property>
            </column>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
</ui>
